py/vttest/local_database.py

```diff
@@ -50,9 +54,14 @@ def vtgate_addr(self):
 
   def config(self):
     """Returns a dict with enough information to be able to connect."""
-    return {
-        'port': vt_processes.vtgate_process.port,
-        }
+    if self.mysql_only:
+      return {
+          'socket': self.mysql_db.unix_socket(),
```

Comment:
this is not going to work in google3, I think you need to let the mysql_db return whatever it wants, and just export it here. You also probably want to export the username, password, and whatever else makes sense.